data-raw/covid19_sa.R

```diff
@@ -0,0 +1,21 @@
+## code to prepare `covid_sa` dataset
+
+library(dplyr)
+library(lubridate)
+
+# Link to data
+data_url <- "https://raw.githubusercontent.com/dsfsi/covid19za/master/data/covid19za_timeline_confirmed.csv" # nolint: line_length_linter.
+
+# Read the data in using the url
+covid19_sa <- read.csv(data_url)
+
+# Clean and subset the data we need
+covid19_sa <- covid19_sa %>%
+  select(date) %>%
+  mutate(date = lubridate::dmy(date)) %>%
+  filter(date <= min(date) + lubridate::days(15)) %>%
+  group_by(date) %>%
+  summarise(cases = n()) %>%
+  ungroup()
+
+usethis::use_data(covid19_sa, overwrite = TRUE)
```

Comment:
When I run this script I get a different data frame from the one currently in the package. I get a 14x2 df, whereas the one saved in the package is 7x2.